web/themes/new_weather_theme/templates/layout/off-canvas-page-wrapper.html.twig

```diff
@@ -0,0 +1,26 @@
+{#
+/**
+ * @file
+ * Default theme implementation for a page wrapper.
+ *
+ * For consistent wrapping to {{ page }} render in all themes. The
+ * "data-off-canvas-main-canvas" attribute is required by the off-canvas dialog.
+ * This is used by the core/drupal.dialog.off_canvas library to select the
+ * "main canvas" page element as opposed to the "off canvas" which is the dialog
+ * itself. The "main canvas" element must be resized according to the width of
+ * the "off canvas" dialog so that no portion of the "main canvas" is obstructed
+ * by the off-canvas dialog. The off-canvas dialog can vary in width when opened
+ * and can be resized by the user. The "data-off-canvas-main-canvas" attribute
+ * cannot be removed without breaking the off-canvas dialog functionality.
+ *
+ * Available variables:
+ * - children: Contains the child elements of the page.
+ *
+ * @ingroup themeable
+ */
+#}
+{% if children %}
+  <div class="dialog-off-canvas-main-canvas height-full" data-off-canvas-main-canvas>
+    {{ children }}
+  </div>
+{% endif %}
```

Comment:
Ooh, what's this file?

Reply:
As far as I can tell, Drupal offers (modal?) dialog capabilities, and this `div` wraps the main page content so that the dialog can distinguish it. 
  
Because I've been using USWDS [utility classes](https://designsystem.digital.gov/utilities/) where possible rather than editing a stylesheet, I needed to be able to edit the attributes of this div. It turns out [this file](https://api.drupal.org/api/drupal/core%21modules%21system%21templates%21off-canvas-page-wrapper.html.twig/10) is part of the standard theme structure for Drupal, and by adding a custom version to our theme I'm simply overriding the default. Btw, the default is nearly identical to this file, with the exception of the `height-full` class on the div. Perhaps there's a better way to accomplish the same thing...

Reply:
> Perhaps there's a better way to accomplish the same thing...

Claire once told me that when working with Drupal, the way that works is the right way. So... there you go. :joy: